src/Client/InvoiceCheckoutOptions.php

```diff
@@ -38,14 +38,18 @@ class InvoiceCheckoutOptions
     /** @var string */
     protected $defaultLanguage;
 
+    /** @var bool */
+    protected $requiresRefundEmail;
+
     public static function create(
         ?string $speedPolicy,
         ?array $paymentMethods,
         ?int $expirationMinutes,
         ?int $monitoringMinutes,
-        ?int $paymentTolerance,
+        ?float $paymentTolerance,
         ?string $redirectURL,
         ?bool $redirectAutomatically,
+        ?bool $requiresRefundEmail,
```

Comment:
Also breaking change, for making it not break anything we would need to put it last and set default value. I would put it at least to last position so that behaviour does not silently change. I will put it in the 2.1.0 release notes and/or add it to 2.0.0 release notes breaking changes to not surprise anybody.

Reply:
Alright I did it, let me know if I did it right

Reply:
As we moved it to v3 you could remove the default value again, but would keep it at last place to keep the previous versions order of params 👍